registration/models.py

```diff
@@ -142,6 +142,7 @@ def resend_activation_mail(self, email, site, request=None):
             profile = self.get(user__email=email)
         except ObjectDoesNotExist:
             return False
+        # TODO: Catch multiple objects returned exception?
```

Comment:
Yes, I added this todo because I don't think email is a necessarily a unique field on user. Do you think it will always be unique? If there are multiple users with the same email address, should they all be resent an activation? If we want to change this to a filter instead of a get, I can do that in a separate PR.